common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -120,6 +126,23 @@ def _cohort_count(self, label):
         """
         return int(label.split(' (')[1].split(')')[0])
 
+    def wait_for_cohort_management_section(self):
+        """
+        Wait for Cohort Management Section to appear.
+        """
+        self.wait_for_element_visibility(self._bounded_selector('#cohort-select'), "Select Cohort is Visible")
+
+    def save_cohort_settings(self):
+        """
+        Click on Save button shown after click on Settings tab or when we add a new cohort.
+
+        NOTE: In case of updating the existing cohort settings, Page will Reload when we click on Save button.
```

Comment:
@muhammad-ammar and I talked about this this morning, and we're going to remove the page refresh and update the view dynamically. I've updated the acceptance criteria of TNL-181.